File: MemoriaDinamica.java
import java.util.ArrayList;

public class MemoriaDinamica {
    public static void main(String args[]) {
        ArrayList<String> frutas = new ArrayList<String>();

        frutas.add("Mango");
        frutas.add("Manzana");
        frutas.add("Banana");
        frutas.add("Uvas");
        System.out.println(frutas);

        frutas.remove(0);
        frutas.remove(1);
        
        frutas.add("Sandía");
        System.out.println(frutas);
    }
}

File: MemoriaDinamica.py
def main():
    frutas = []

    frutas.append("Mango")
    frutas.append("Manzana")
    frutas.append("Banana")
    frutas.append("Uvas")
    print(frutas)

    frutas.pop(0)
    frutas.pop(1)
    
    frutas.append("Sandía")
    print(frutas)

if __name__ == "__main__":
    main()
